nixos/modules/services/hardware/spacenavd.nix

```diff
@@ -13,12 +13,14 @@ in {
   };
 
   config = mkIf cfg.enable {
-    systemd.user.services.spacenavd = {
+    systemd.user.services.spacenavd = mkIf (!versionAtLeast pkgs.spacenavd.version "1.1") {
```

Comment:
I'm not a NixOS expert, but when you update `pkgs.spacenavd` you could skip this fallback. Or what's the idea behind this fallback?

Reply:
The idea is to be able to still use the module with old versions of spacenavd, e.g. if you have an overlay with a custom, patched spacenavd. Might be overkill though.

Reply:
I removed the fallback. People updating to the new module will probably also use the new package.